src/scratchtocatrobat/tools/logger.py

```diff
@@ -45,7 +46,9 @@ def setup_logging():
     log.setLevel(logging.DEBUG)
 
     log_dir = helpers.config.get("PATHS", "logging")
-    fh = logging.FileHandler(os.path.join(log_dir, "s2cc-{}.log".format(datetime.now().isoformat().replace(":", "_"))))
+    global log_file
+    log_file = os.path.join(log_dir, "s2cc-{}.log".format(datetime.now().isoformat().replace(":", "_")))
```

Comment:
Should we change the location we save logfiles to? Right now it's the same folder the converter uses.